terraform/install.sh

```diff
@@ -39,18 +43,12 @@ applyTerraform() {
 
 getExternalIp() {
   external_ip=""; 
-  while [ -z $external_ip ]; 
-  do
+  while [ -z $external_ip ]; do
      log "Waiting for Hipster Shop endpoint..."; 
      external_ip=$(kubectl get svc frontend-external --template="{{range .status.loadBalancer.ingress}}{{.ip}}{{end}}"); 
      [ -z "$external_ip" ] && sleep 10; 
   done; 
-
```

Comment:
I see you remove the curl verification for the external IP. Why?
kubectl get service won't provide this.

Reply:
If I'm reading it right, the old code would only display Hipstershop's service IP if it was reachable, and just silently continue it if it's not. Is that a useful check? I think we should display the URL to the user matter what, so the check seems a bit unnecessary.

Reply:
The idea is to check that the address is accessible and if it is - display "success" and the address to the user. If it's not accessible - we should indicate the failure.

Reply:
Do you think we should completely cancel deployment if the curl verification fails (after a number of retries of course)? Or is there anything we can do to fix any possible issues as part of the script? As it is, I don't think anyone even would even notice the success message among the logs, and nothing happens on error

Reply:
No need to cancel the deployment.
We can just log it and in subsequent flow indicate to the user that external IP of the hipster shop cluster is not available. Even though there might be more logs that will follow this, if the user will want to troubleshoot the problem and share what happened with us  - log like that can be useful.

Reply:
ok, I added it back with logs for either result